app/controllers/search_controller.rb

```diff
@@ -50,7 +50,7 @@ def search
         end
 
         if depth == 'instant'
-          results.each { |x| x[:image] = x[:image].url(:small) }
+          results.each { |x| x[:image] = x[:image].url(:medium) }
```

Comment:
This happened because not all types have `:small` sizes.  We should probably remedy this and standardize somewhat.

Until we get around to that, this'll work